presto-orc/src/main/java/com/facebook/presto/orc/OrcRecordReader.java

```diff
@@ -595,6 +614,49 @@ else if (type.getOrcTypeKind() == OrcTypeKind.MAP) {
         return statistics.build();
     }
 
+    /**
+     * @return All StreamReaders' local buffer sizes, plus all instance sizes
```

Comment:
`@return the size of memory retained by all the stream readers (local buffers + object overhead)`